fix: ramp slider (#829)

bernhard-adobe opened the pull request:
Adding back ramp-slider and removing color-slider.


## Description
Addresses issue #829 

- added back ramp-slider
- removed color-slider from slider page that is now it's own component under [/docs/colorslider.html](/docs/colorslider.html)


## How and where has this been tested?
URL: http://localhost:3000/docs/slider.html
Chrome Version 85.0.4183.102 (Official Build) (64-bit) on MacOS


## Screenshots
<!-- If applicable, add screenshots to show what you changed -->

![Screen Shot 2020-09-22 at 13 16 22](https://user-images.githubusercontent.com/52184321/93917037-6cfee100-fcd8-11ea-9261-7e64df94489b.png)

## To-do list
<!-- Put an "x" to indicate you've done each of the following -->
- [x] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).
<!-- If this pull request isn't ready, add any remaining tasks here -->
- [x] This pull request is ready to merge.

lazd reviewed (changes requested):
Commit message needs to include:

```
BREAKING CHANGE: Slider color variant removed, used Color Slider component
```

Also need migration guide.

bernhard-adobe commented:
Good points @lazd .
I added a migration guide. Unfortunately your suggested commit message is not working because of the commit rules with uppercases:
```
$ g commit -m "BREAKING CHANGE: Slider color variant removed, used Color Slider component"
husky > commit-msg (node v12.16.2)
⧗   input: BREAKING CHANGE: Slider color variant removed, used Color Slider component
✖   header must not be longer than 72 characters, current length is 74 [header-max-length]
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 3 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint

```

I used instead `g commit -m "fix: breaking change: slider color variant moved to color slider. (#850)"`
I also added the breaking-change label.

GarthDB commented:
The disabled color seems to have changed for ramp slider. Is that a result of the tokens changing?

lazd commented on components/slider/metadata/slider.yml:
```diff
@@ -1,5 +1,11 @@
+sections:
+  - name: Migration Guide
+    description: |
+      ### Color Slider moved  out of Sliders into it's own component
```
```suggestion
      ### Color slider is now a separate component
```

lazd commented on components/slider/metadata/slider.yml:
```diff
@@ -1,5 +1,11 @@
+    description: |
+      ### Color Slider moved  out of Sliders into it's own component
+      Color Slider moved to [Color slider](colorslider.html).
+      Replace class names starting with `.spectrum-Slider` to `.spectrum-ColorSlider`.
```
```suggestion
      Replace class names starting with `.spectrum-Slider` with `.spectrum-ColorSlider`.
```
